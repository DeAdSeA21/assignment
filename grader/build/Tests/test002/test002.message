Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t39.11
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
