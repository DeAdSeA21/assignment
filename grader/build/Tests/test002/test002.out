Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	95.44
2	90.86
3	86.25
4	81.63
5	76.99
6	72.32
7	67.64
8	62.94
9	58.21
10	53.47
11	48.70
12	43.91
